metadata-schema/closureSchemaOpen.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$id": "/schema/open-closure",
+  "type": "object",
+  "allOf": [
+    {
+      "if": {
+        "properties": {
+          "closure_type": {
+            "enum": [
+              "open_on_transfer",
+              "Open",
+              "open",
+              "OPEN"
+            ]
+          }
+        }
+      },
+      "then": {
+        "properties": {
+          "closure_start_date": {
+            "type": "null"
+          },
+          "closure_period": {
+            "type": "null"
+          },
+          "foi_exemption_code": {
+            "type": "null"
+          },
+          "foi_exemption_asserted": {
+            "type": "null"
+          },
+          "title_alternate": {
+            "type": "null"
+          },
+          "description_alternate": {
+            "type": "null"
+          },
+          "title_closed": {
```

Comment:
this should be a boolean type - as in base schema. The csv utiils will have converted the No in the csv to a false

Reply:
I'm not sure if the  user is allowed to enter nothing here or has to use NO which is converted to false
better check title_closed and description_closed are null or false

Reply:
Gareth said that we probably won't even show the title_closed/description_closed to the user but since it's still there at the moment I have added a check for if the value is false or null.